ChangeLog.d/add_MBEDTLS_TARGET_PREFIX_to_cmake.txt

```diff
@@ -0,0 +1,6 @@
+Features
+   * Add MBEDTLS_TARGET_PREFIX CMake variable, which is prefixed to the start of
+     the mbedtls, mbedcrypto, mbedx509 and apidoc CMake targets. This can be
+     used by external CMake projects that include this one to avoid CMake
```

Comment:
"to avoid CMake variable name clashes". It is rather to avoid target name clashes,